README.md

```diff
@@ -1,9 +1,8 @@
-# p5-wrapper
+# react-p5-wrapper
 
-This Component lets you integrate p5 Sketches into your React App.
-
-Original Code was from [Ivan Malyugin](https://discuss.reactjs.org/users/IMalyugin) from a [Discussion in the React Forum](https://discuss.reactjs.org/t/using-react-with-p5-js/5565)
+This component lets you integrate p5 sketches into your React App.
 
+Original code was created by Ivan Malyugin.
```

Comment:
I would keep the original line for this since it provided some context and linked to credit.

Reply:
The problem is that such a site no longer exists and the links are dead. So it seems more confusing to me than giving context.

Reply:
I rephrased that sentence.

Reply:
I removed it altogether because if the resource doesn't exist and we can't give true context then it shouldn't be there IMO. I also added a section for yarn and updated the examples a little. Let me know what you think, if you are happy then we can merge this!

Reply:
Agreed, in my recent changes, I've used the original library names in the links and consistently formatted the examples of code and commands.

Thanks @jamesrweb for your input and cooperation!

🚢 🚢 🚢